src/js/server/hcapi/applications.js

```diff
@@ -4,6 +4,23 @@ const { createHttpError } = require('./errors');
 
 const applicationsRouter = new Router();
 
+applicationsRouter.get('/', (req, res, next) => {
+  HackerApplication
```

Comment:
Is there an alternative you'd suggest? We could get all the application IDs, along with some other pieces of information to filter by, but we're likely to request that data for each hacker immediately afterwards anyway, if we want to do the filtering client-side. Probably not all the data is going to be used, though, so we could always crop the entries being returned here a little (like t-shirt size, etc.)?